examples/speech_to_text_v1.py

```diff
@@ -21,3 +23,35 @@
                 timestamps=True,
                 word_confidence=True),
             indent=2))
+
+# Example using websockets
+class MyRecognizeCallback(RecognizeAbstractCallback):
```

Comment:
Do users need to override each method in `RecognizeAbstractCallback`? If that's the case let's make `RecognizeAbstractCallback` a concrete class and leave each method as 

```py
def method(self):
    pass
```